apport/report.py

```diff
@@ -890,6 +890,9 @@ def add_gdb_info(self, rootdir=None, gdb_sandbox=None):
         # limit maximum backtrace depth (to avoid looped stacks)
         gdb_cmd += ["--batch", "--ex", "set backtrace limit 2000"]
 
+        # disable debuginfod
+        gdb_cmd += ["-iex", "set debuginfod enable off"]
```

Comment:
How about combining those two commands into one:
```suggestion
        gdb_cmd += [
            "--batch",
            # limit maximum backtrace depth (to avoid looped stacks)
            "--ex",
            "set backtrace limit 2000",
            "-iex",
            "set debuginfod enable off",
        ]
```